### DISCUSSION TITLE
Stream from Mic and BH to VOIP program (discord/skype etc) and also be able to hear.

### DISCUSSION THREAD

**Discussion by quantuminformation** [ExistentialAudio/BlackHole#211] [General]
My requirements are to be able to stream live the audio from my computer and my mic which I can do with no issue. I also need my friend on skype / discord to be able to hear from my mac what audio comes from BH and my voice. Sorry if my requirements are way over the top here! Atm  I cannot hear any audio through my headphones

I have tried the following:

MIDI setup:
![image](https://user-images.githubusercontent.com/216566/94412649-5219ea00-017a-11eb-8d6c-04b4f566b560.png)

Logic:

![image](https://user-images.githubusercontent.com/216566/94412679-5c3be880-017a-11eb-9107-7b7f86536191.png)
![image](https://user-images.githubusercontent.com/216566/94412713-678f1400-017a-11eb-9264-1edcf980f28c.png)
I'm not to sure what the purpose is of the above ^

Discord:
![image](https://user-images.githubusercontent.com/216566/94412836-93aa9500-017a-11eb-9542-89b7316c7818.png)
mac settings
![image](https://user-images.githubusercontent.com/216566/94412866-9d33fd00-017a-11eb-8850-6eef6be85352.png)
![image](https://user-images.githubusercontent.com/216566/94412892-a58c3800-017a-11eb-93cd-02537748471f.png)

**Comment by quantuminformation** [comment]
I tried to make things simple with this in logic

![image](https://user-images.githubusercontent.com/216566/94471312-3a1d8700-01c9-11eb-8f89-c11a2d710051.png)
![image](https://user-images.githubusercontent.com/216566/94471339-473a7600-01c9-11eb-88a3-e20dc872934b.png)

and I can hear in my headphones, but no audio is picked up in ag as input device in obs

![image](https://user-images.githubusercontent.com/216566/94471395-676a3500-01c9-11eb-9786-bb91aeb8e0ba.png)

**Comment by quantuminformation** [comment]
I have to add them individually to get SL OBS to pick them up 
![image](https://user-images.githubusercontent.com/216566/94471894-33434400-01ca-11eb-96c1-dbc78e44eadb.png)

**Comment by devinroth** [comment]
An aggregate doesn’t output audio to all the outputs. They need to be routed that way. Multi-outputs will do that so you can use a multi-output in OBS instead. Or do what you just did which also works.

Does that make sense?
